src/gui_common/PhotographPreview.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Godot;
+
+/// <summary>
+///   Base class for displaying photograph tasks
+/// </summary>
+/// <remarks>
+///   <para>
+///     To use this, instantiate its scene, extend this class and decide:
+///     <list type="bullet">
+///       <item>
+///         When to update preview by calling UpdatePreview()
+///       </item>
+///       <item>
+///         What to preview by overriding SetupImageTask()
+///       </item>
+///     </list>
+///   </para>
+/// </remarks>
+public abstract class PhotographPreview : Control
```

Comment:
```suggestion
public abstract class PhotographablePreview : Control
```
I'd rename it like that to make it clear that this is not for any "photo" previews but previews for things the game considers "photographable"